public/docs/ts/latest/guide/index.jade

```diff
@@ -30,85 +22,68 @@ table(width="100%")
     td
       :marked
         In-depth analysis of Angular features and development practices.
   tr(style=top)
-    td <b><a href="../cookbook">Cookbook</a></b>
+    td <b><a href="../cookbook/">Cookbook</a></b>
     td
       :marked
         Recipes for specific application challenges, mostly code snippets with a minimum of exposition.
 
 :marked
-  ## Learning path
-  
-  You don't have to read the guide straight through.  Most pages stand on their own.
-
-  For those new to Angular, the recommended learning path runs through the *Guide* section:
-
-  1. For the big picture, read the [Architecture](architecture.html) overview.
-
-  1. Try [QuickStart](../quickstart.html). QuickStart is the "Hello, World" of Angular.
-  It shows you how to set up the libraries and tools you'll need to write *any* Angular app.
-
-  1. Take the *Tour of Heroes* [tutorial](../tutorial), which picks up where QuickStart leaves off,
-  and builds a simple data-driven app. The app demonstrates the essential characteristics of a professional application:
-  a sensible project structure, data binding, master/detail, services, dependency injection, navigation, and remote data access.
-
-  1. [Displaying Data](displaying-data.html) explains how to display information on the screen.
+  A few early pages are written as tutorials and are clearly marked as such.
+  The rest of the pages highlight key points in code rather than explain each step necessary to build the sample.
+  You can always get the full source through the #{_liveLink}s.
 
-  1. [User Input](user-input.html) covers how Angular responds to user behavior.
+:marked
+  ## Code samples
 
-  1. [Forms](forms.html) handles user data entry and validation within the UI.
+  Each page includes code snippets that you can reuse in your applications.
+  These snippets are excerpts from a sample application that accompanies the page.
 
-  1. [Dependency Injection](dependency-injection.html) is the way to build large, maintainable applications
-  from small, single-purpose parts.
+block example-links
+  :marked
+    Look for a link to a running version of that sample near the top of each page,
+    such as this <live-example name="architecture"></live-example> from the [Architecture](architecture.html) page.
 
-  1. [Template Syntax](template-syntax.html) is a comprehensive study of Angular template HTML.
+    The link launches a browser-based, code editor where you can inspect, modify, save, and download the code.
 
-  After reading the above sections, you can skip to any other pages on this site.
+:marked
+  ## Reference pages
 
-  ## Code samples
+  The [Cheat Sheet](cheatsheet.html) lists Angular syntax for common scenarios.
 
-  Each page includes code snippets that you can reuse in your applications.
-  These snippets are excerpts from a sample application that accompanies the page.
+  The [Glossary](glossary.html) defines terms that Angular developers should know.
```

Comment:
You didn't like the compactness of a bulleted list for all the "reference page" entries?

Actually, same question for the feedback entries?